tools/coqdep/lib/common.ml

```diff
@@ -8,18 +8,10 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-open Format
-open Lexer
-
-let (//) = System.(//)
```

Comment:
This was included before, since people wanted `//` to appear throughout.

Reply:
Writing System.(...) everywhere is not really a readability gain IMO.

Reply:
Maybe it is not a gain, but that op reexporting was worse IMHO. Also, the semantics of (//) are messy so indeed hopefully we can get rid of that function altogheter in the code, down from the 4 current ocurrences. We shouldn't have that kind of code in `System` to start with.

Reply:
At some point I tried to introduce a `Path.t` type, but that's a lot of work. At least, `Filename.concat` has clear semantics and it is a standard library fucntion so people knows it, vs our own hand-rolled version that we don't know what the invariants are.

Reply:
Actually at some point there were several copies of `let (//) = ` in Coq, now we are down to only two, so indeed I feel better by forcing people to qualify it.

Reply:
@ejgallego OK, then replacing `(//)` with `Filename.concat` is a good idea IMO. I'm sure it is slightly more readable too.

Reply:
OK, that is unfortunate. For now let's keep it how it is then.

Reply:
>  but that op reexporting was worse

Note that this can be avoided:
```ocaml
open struct let (//) = System.(//) end
```